klippa/tests/integration_test.rs

```diff
@@ -0,0 +1,538 @@
+//! Test subsetter output against expected results.
+//!
+//! This reads test configs from harfbuzz subset test suite,
+//! generate a group of tests to perform, run and then compare the output against the stored expected result
+//!
+//! To generate the expected output files, pass GEN_EXPECTED_OUTPUTS=1 as an
+//! environment variable.
```

Comment:
is this something we want to do? I would expect that the expected outputs in harfbuzz to be canonical?

Reply:
yes, we'd like to be able to generate our own golden files. Because currently we only have the most basic subset support and nothing matches harfbuzz results.